projects/angular/src/emphasis/alert/utils/alert-types.ts

```diff
@@ -4,5 +4,9 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-// @TODO Make this an enum
-export const ALERT_TYPES: string[] = ['info', 'warning', 'danger', 'success'];
+export enum AlertType {
```

Comment:
That's something I believe we need to avoid. While it is good for type checker, declaring the AlertType in your component so you can use it in the template its kinda bit too much.

What about if we use `type` instead of `enum`? We still keep the type checker working but we don't force people to use an enum

```
 export type AlertType = 'info' | 'warning' | 'danger' | 'success';
```